src/Microsoft.Health.Fhir.CosmosDb/Features/Search/Queries/QueryBuilder.cs

```diff
@@ -134,6 +158,30 @@ private void AppendFilterCondition(string name, object value)
                         .AppendLine(_queryParameterManager.AddOrGetParameterMapping(value));
             }
 
+            private void AppendArrayContainsFilter(string logicalOperator, params (string, string)[] conditions)
```

Comment:
don't really need this method since there is only one condition anyways

Reply:
I added this so that if we have multiple array property values in the index, then we can use this method, sure currently we have only one. I plan to keep it this way unless you feel strongly.